OJP/OJP_Locations.xsd

```diff
@@ -12,17 +12,17 @@
 		<xs:documentation>FUNCTION 4: Coordinate to Address</xs:documentation>
 		<xs:documentation>All functions integrated into one request / response</xs:documentation>
 	</xs:annotation>
-	<xs:group name="PlaceInformationRequestGroup">
+	<xs:group name="LocationInformationRequestGroup">
 		<xs:sequence>
 			<xs:choice>
-				<xs:element name="InitialInput" type="InitialPlaceInputStructure">
+				<xs:element name="InitialInput" type="InitialLocationInputStructure">
 					<xs:annotation>
-						<xs:documentation>Initial input for the place information request. This input defines what is originally looked for.</xs:documentation>
+						<xs:documentation>Initial input for the location information request. This input defines what is originally looked for. Be aware that this also can contain PLACEs.</xs:documentation>
 					</xs:annotation>
 				</xs:element>
 				<xs:element name="PlaceRef" type="PlaceRefStructure">
 					<xs:annotation>
-						<xs:documentation>Place for further refinement. If a place in a previous response was marked as not "complete" it can be refined by putting it here.</xs:documentation>
+						<xs:documentation>LOCATION / PLACE for further refinement. If a location/place in a previous response was marked as not "complete" it can be refined by putting it here.</xs:documentation>
```

Comment:
```suggestion
						<xs:documentation>LOCATION for further refinement. If a location/place in a previous response was marked as not "complete" it can be refined by putting it here.</xs:documentation>
```